core/controllers/dashboard_test.py

```diff
@@ -66,9 +66,7 @@ def test_logged_in_notifications_dashboard(self):
         self.logout()
 
 
-class MyExplorationsHandlerTest(test_utils.GenericTestBase):
-
-    MY_EXPLORATIONS_DATA_URL = '/myexplorationshandler/data'
+class DashboardHandlerTest(test_utils.GenericTestBase):
```

Comment:
To better tighten our naming, can we maybe do ``CreatorDashboard`` instead of just ``Dashboard``, here and elsewhere? We may have other dashboards in the future and it would  be nice to avoid the (exploration) editor/collection editor situation.